lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -45,11 +46,15 @@ def ask_question_loop
     end
 
     def activate
+      if key_exist? && force
+        backup_key
+      end
+
```

Comment:
well, this scenario needs to do more changes. I need to check if the `File.cp` for backup success, if not, ask user whether to continue or not (and suggest user do a manual backup before continue)